src/backend/distributed/executor/local_executor.c

```diff
@@ -242,15 +251,20 @@ ExtractParametersForLocalExecution(ParamListInfo paramListInfo, Oid **parameterT
  * distributed table, meaning it is part of distributed execution.
  */
 static void
-LogLocalCommand(Task *task)
+LogLocalCommand(char *queryString)
```

Comment:
This function no longer checks whether it should log local commands, unlike LogRemoteCommands. That seems confusing.

Reply:
Maybe we can separate the check from LogRemoteCommands as well but that would mean every time we call LogRemoteCommands we would need to call the function that does the check as well. So I am also neutral to this change. Feel free to close this PR if you want to.